tasks-by-users/nehalskdev/01 Project/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<title> Kraj Men's wear </title>
```

Comment:
why this title tag is in html @nehalskdev

Reply:
my apologies, i have placed it in right manner now